app/controllers/notes_controller.rb

```diff
@@ -1,8 +1,20 @@
 class NotesController < ApplicationController
   include ActionView::Helpers::TextHelper
+  DEFAULT_NOTES_PER_PAGE = 15
   PAGES_PER_SCREEN = 20
 
+  def index
+    filtered_notes
+
+    render partial: 'table' if request.xhr?
```

Comment:
Ideally we use turbo_streams here, but that will have  to wait until upgrade.

But this is similar pattern, as if we are doing format.html, and format.json, but in this case, if request.xhr we will render just the partial that we will use to replace the table contents.